### PR TITLE
Persistence service

### PR DESCRIPTION
**Pull request by telebovich**
Create a hardcoded machzor in IndexedDB

### CODE REVIEW

File: src/app/_services/persistence.service.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        autoIncrement: true 
+      });
+
+      objectStore.createIndex("start", "start", { unique: true });
+
+      objectStore.transaction.oncomplete = (event) => { }
+    }
+  }
+  
+  public addPeriod(period: Period) {
```
Comment (Israel-Kli) [inline src/app/_services/persistence.service.ts:32]: @telebovich 

1. I see you don't call this method anywhere... Is it for the future purpose?
2. Why you called it addPeriod while in other places it's named `addMachzor`?

**Review by Israel-Kli — COMMENTED** [review]
Tzvika, thank you for this PR!! :1st_place_medal: 
I'll try to checkout and play with it a bit.
Only one question for now...

File: src/app/_services/persistence.service.ts (hunk 1 above)
Comment (telebovich) [reply src/app/_services/persistence.service.ts:32]: I started playing with API then I found a wrapper for the API. I will refactor the code a little bit.

**Review by Israel-Kli — APPROVED** [review]

File: src/app/_services/persistence.service.ts (hunk 1 above)
Comment (Israel-Kli) [reply src/app/_services/persistence.service.ts:32]: I merged that, I'll try to find time to play with it better...

**Pull request merged**